cmake_format/formatter.py

```diff
@@ -1206,7 +1207,7 @@ def assert_lt(self, cursor):
 
   def forge_cursor(self, cursor):
     self._cursor = cursor
-
+    
```

Comment:
@MaximRybak I suppose that formating existing python code is not a good idea.
I can see it not the first time. Please take care about changes in your commits.